spec/rails_helper.rb

```diff
@@ -28,10 +28,8 @@ module Features
   # Prevent leaking view contexts between tests
   # see https://github.com/drapergem/draper/issues/814
   # see https://github.com/drapergem/draper/issues/655
-  [:decorator, :controller, :mailer].each do |type|
-    config.before(:each, type: type) { Draper::ViewContext.clear! }
-    config.after(:each, type: type) { Draper::ViewContext.clear! }
-  end
+  config.before { Draper::ViewContext.clear! }
```

Comment:
Maybe this will fix #1854 and #3467???

Reply:
:pray: This was my hope.

Reply:
I'm so hopeful i'm gonna add it to the PR description to auto-close them.

Reply:
I'm hopeful as well: https://github.com/drapergem/draper/issues/523